services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java

```diff
@@ -161,33 +169,52 @@ private boolean isEfgsEnforceDateBasedDownloadAndNotSeen(String batchTag) {
 
   private Optional<String> processBatchAndReturnNextBatchId(
       FederationBatchInfo batchInfo, FederationBatchStatus errorStatus) throws FatalFederationGatewayException {
+    // Branch 0
+    bakfcCovVisit(0);
+
     LocalDate date = batchInfo.getDate();
     String batchTag = batchInfo.getBatchTag();
     logger.info("Processing batch for date {} and batchTag {}.", date, batchTag);
     AtomicReference<Optional<String>> nextBatchTag = new AtomicReference<>(Optional.empty());
     try {
+      // Branch 1
+      bakfcCovVisit(1);
       BatchDownloadResponse response = federationGatewayDownloadService.downloadBatch(batchTag, date);
       AtomicBoolean batchContainsInvalidKeys = new AtomicBoolean(false);
       nextBatchTag.set(response.getNextBatchTag());
       response.getDiagnosisKeyBatch().ifPresentOrElse(batch -> {
```

Comment:
This seems like a (slightly hidden) if-else statement, so branch visits should be inserted in the next line and into line 208, `() -> logger.info("Batch for date {} and batchTag {} did not contain any keys.", date, batchTag))`

(This branch was probably not detected by lizard either)